README.md

```diff
@@ -38,21 +40,21 @@ $ fun -h
     build               Build the dependencies
 ```
 
-## 使用
+## Usage
 
-安装完命令行工具之后，即可开始进行代码的开发了。为了配合 fun 工具，您需要创建一个工程目录，然后在工程目录下创建一个 `template.yml` 模板文件，fun 会将该目录视为一个工程。
+Once you have installed the command line tools, you are ready to write code. In order to use fun tool, you need to create a project directory and then create a file named template.yml in the project directory. Fun will treat the directory as a project.
```

Comment:
After installed, you can go ahead to develop. In order to use `fun`,  you need to create a project directory which contains a file named `template.yml`.